views/nodes/show.erb

```diff
@@ -3,25 +3,24 @@
     <a href="/node/<%= @node.id %>/edit">edit</a>
   </h2>
   <p>
-    Project Id: <%= @node.project_id %> <br>
```

Comment:
I think that the Project Id is something that should still be displayed in the views